src/lexers/jsx-lexer.js

```diff
@@ -69,6 +70,34 @@ export default class JsxLexer extends JavascriptLexer {
         entry.namespace = namespace
       }
 
+      const getCustomAttrs = (node, opts) => {
```

Comment:
Well code is not real sexy, but I did it real quick. let me know if you have a better idea.
Also related to how we offer to pass the option to the parser.

In general in case you wonder: jsx parser was the only parser who wasn't automatically extracting custom props.